subt/docker/robotika/ros/robotika/src/drone_height.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+import rospy
+from nav_msgs.msg import *
+from std_msgs.msg import *
+from sensor_msgs.msg import *
+from geometry_msgs.msg import *
+
+import pdb
+    
+HEIGHT = 1.5
```

Comment:
DESIRED_HEIGHT_ABOVE_GROUND?

If I learned anything from depth processing and obstacle avoidance ... let's make everything externally configurable from the beginning as much as possible. The reason you cannot easily use the depth camera module is exactly because such constants and parameters are hard-coded.